Powershell/InstallHuntress.powershellv2.ps1

```diff
@@ -784,10 +803,11 @@ function logInfo {
 # In the past we copied to the users temp folder, difficult to find on machines with lots of profiles. Solved this by always placing the log in the normal Huntress folder.
 function copyLogAndExit {
     Start-Sleep 1
-    $logLocation = $DebugLog
-    $agentPath   = getAgentPath
-    Copy-Item -Path $logLocation -Destination $agentPath -Force
-    Write-Output "$($Debuglog) copied to $(getAgentPath)"
+    $agentPath = getAgentPath
+    $logLocation = Join-Path $agentPath "HuntressPoShInstaller.log"
+    if (!(Test-Path -path $agentPath )) {New-Item $agentPath  -Type Directory}
```

Comment:
sorry going to nitpick again 😅 spacing after $agentPath, in both the condition and the function, condition has trailing space, function has two spaces instead of one

Reply:
Corrected the spacing :)